packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -952,7 +952,7 @@ export const DEFAULT_TOKEN_ADJACENCY_GRAPH_BY_CHAIN_ID = valueByChainId<TokenAdj
                 builder
                     .add(AVALANCHE_TOKENS.aWETH, AVALANCHE_TOKENS.nETH)
                     .add(AVALANCHE_TOKENS.nETH, AVALANCHE_TOKENS.aWETH);
-                //Trader Joe MAG/MIM pool
+                // Trader Joe MAG/MIM pool
```

Comment:
```suggestion
                //Trader Joe MAG/MIM pool
```
Just saw this is getting fixed inadvertently  another PR